src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, useEffect, useState } from 'react';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+enum ErrorType {
+  NoPeople = 'There are no people on the server',
```

Comment:
I think that it is not an error it's just because the resolve is Ok, but there are no people in array